### ISSUE TITLE
vg_create is not idempotent 

### ISSUE THREAD

**Issue by hunter86bg** [gluster/gluster-ansible-infra#132]
For some reason vg_create is using a command that fails if you need to rerun the setup:
 https://github.com/gluster/gluster-ansible-infra/blob/6ce78857a1dc992364c8a5b564ca9b09c89bad9e/roles/backend_setup/tasks/vg_create.yml#L65

We need to use idempotent modules where possible

**Comment by sac** [comment]
It was idempotent earlier, but the commit https://github.com/gluster/gluster-ansible-infra/commit/0c1f454d98 changed that. But I do not see in the commit message why it was changed.

**Comment by hunter86bg** [comment]
@sac ,
I was wondering if that was on purpose or not.
I can open a PR, but most probavly we need a way to setup Molecule for the roles in this repo.

Currently, a lot of oVirt users report issues with this one (for example something failed later and when rerunning it complains the VG exists).

**Comment by hunter86bg** [comment]
Also, I don't see peer review on that commit . How is that possible?

**Comment by sac** [comment]
I don't think you should spend time on the PR. The committer should revert his changes unless he has a good reason. We can ask the maintainers to take a look.

@rchikatw @gobindadas can you please take a look at this issue? Also, why did a non-idempotent CLI replace the module?

**Comment by rchikatw** [comment]
@sac @hunter86bg 
Hello,
ansible-core 2.12 introduced from RHV 4.4 SP1 release and ansible-core no longer uses the gluster.gluster and community.general ansible collection. Because of this change, I have posted the patches to remove these gluster modules to CLI.

**Comment by hunter86bg** [comment]
@rchikatw 
 we can't provide the modules with a separate rpm ?

**Comment by gobindadas** [comment]
This is not the issue only for RHV, it's also issue for centos stream which has ansible-core. So it's a problem for both upstream and downstream.

**Comment by gobindadas** [comment]
So basically there are two ways to deal with it.
1- Change all the modules to command/shell
2- Convert all the modules into ansile collection and use those (Which need lot of effort and process)
So we decided to go with command line approach for quicker fix.

**Comment by hunter86bg** [comment]
@gobindadas ,
I have never done an Ansible collection before but at least I can take a look if it's in my capabilities to do it.
